src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserController.java

```diff
@@ -76,39 +82,84 @@
 
-        if (!StringUtils.isEmpty(email)) {
-            log.info("email not empty");
-            if (isRolePuiUserManager) {
-                profUsersEntityResponse = retrieveUserByEmail(email);
-            } else {
-                throw new AccessDeniedException("403 Forbidden");
+        if (!isRolePuiUserManager) {
+            if (StringUtils.isEmpty(status)) {
+                status = "Active";
             }
-        } else {
+            profExtUsrReqValidator.validateStatusIsActive(status);
+        }
+
+        profUsersEntityResponse = searchUsersByOrganisation(organisationIdentifier, showDeleted, true, status, page, size);
 
-            if (!isRolePuiUserManager) {
-                if (StringUtils.isEmpty(status)) {
-                    status = "Active";
-                }
-                profExtUsrReqValidator.validateStatusIsActive(status);
+        return profUsersEntityResponse;
+    }
+
+    @ApiOperation(
+            value = "Retrieves the user with the given email address if organisation is active or showDeleted flag ",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
             }
+    )
+    @ApiParam(
+            name = "email",
+            type = "string",
+            value = "The email of the desired user to be retrieved",
+            required = false
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "A professional user along with their details",
+                    response = ProfessionalUsersEntityResponse.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "An invalid email was provided"
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = "Invalid authorization"
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "No user was found with the provided email address"
+            )
+    })
+    @GetMapping(
+            value = "/user",
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE
+    )
+    @Secured({"pui-user-manager"})
+    public Optional<ResponseEntity> findUserByEmail(@ApiParam(hidden = true) @OrgId String organisationIdentifier,
+                                                    @ApiParam(name = "email", required = false) @RequestParam(value = "email", required = false) String email) {
+
+        Optional<ResponseEntity> optionalResponseEntity;
 
-            profUsersEntityResponse = searchUsersByOrganisation(organisationIdentifier, showDeleted, true, status, page, size);
+        if (isValidEmail(email)) {
+            log.info("email is valid");
+            optionalResponseEntity = Optional.ofNullable(retrieveUserByEmail(email));
+        } else {
+            throw new InvalidRequest("The email provided is invalid");
```

Comment:
The numbered exception enum has a rather generic response, I have instead changed the message to "throw new InvalidRequest("The email provided '" + email + "' is invalid");" 
- so it will now include the email and still throw 400 bad request.